pandas/tests/indexes/common.py

```diff
@@ -37,7 +37,7 @@ def verify_pickle(self, indices):
     def test_pickle_compat_construction(self):
         # this is testing for pickle compat
         if self._holder is None:
-            return
```

Comment:
is this actually hit?

Reply:
I don't know, I didn't write these tests, and the inner workings of `indexes/common.py` are not immediately apparent (which files call it, what do they fill `indices` with, etc.).

Reply:
Happy to leave the bare return in there, if that's preferred.

Reply:
well, put a breakpoint there and see if you would.

Reply:
I'm not able to do work on any of this for 2 weeks now. I'll have a look after.